docs/src/api.md

```diff
@@ -1,5 +1,5 @@
 # ApproximateGPs API
 
 ```@autodocs
-Modules = [ApproximateGPs]
+Modules = [ApproximateGPs, ApproximateGPs.API, ApproximateGPs.SparseVariationalApproximationModule, ApproximateGPs.LaplaceApproximationModule]
```

Comment:
The only potential downside I would see is that it might look a bit weirder to see the fully-qualified names in the API docs (see https://juliagaussianprocesses.github.io/ApproximateGPs.jl/previews/PR96/api/)

Reply:
On the other hand, that keeps related functions next to each other, and perhaps it would actually be more helpful to have different sub-pages for each of the approximations?

Reply:
I really like the idea of having one page per approximation.